android/app/build.gradle

```
@@ -56,7 +56,31 @@ project.ext.react = [
     enableHermes : true,
 ]
 
+def signingDebug = getSigningConfigs()['debug']
+def signingRelease = getSigningConfigs()['release']
+
 android {
+    if (signingDebug || signingRelease) {
+        signingConfigs {
```

Maybe something like this to reduce the number of times we parse `app.json`?
```suggestion
project.ext.signingConfigs = getSigningConfigs()

android {
    def signingDebug = project.ext.signingConfigs['debug']
    def signingRelease = project.ext.signingConfigs['release']
    if (signingDebug || signingRelease) {
        signingConfigs {
```